build-support/check_github_labels.py

```diff
@@ -7,17 +7,19 @@
 '''
 import json
 import os
+from os import environ, getenv
 import re
 import sys
 import urllib.request
 
-
 def get_pr_num():
     match = re.match(r'.*terrier_PR-([^@/]*).*', os.getcwd())
     if match and len(match.groups()) == 1:
         return int(match.groups()[0])
     return None
 
+def is_correct_build_folder():
+    return getenv('JOB_NAME').startswith('terrier/')
```

Comment:
It should be `noisepage` not `terrier`? Or is this a Jenkins thing?

Reply:
Yah, this is a Jenkins thing. If we rename the Jenkins job this will need to change. I created this cause any build from the testing team folder wouldn't pass this first step. Cause it would try to lookup the PRs in our repo on Noisepage.